tests/core/Did.spec.ts

```diff
@@ -0,0 +1,8 @@
+import Did from '../../lib/core/Did';
+
+describe('DID', async () => {
+  it('isDid() should return false if DID method is not in the DID.', async () => {
+    const isDid = Did.isDid('did:abcdefg');
+    expect(isDid).toBeFalsy();
+  });
```

Comment:
could add a test for a string that fails the first condition of isDid() like 'abcdefg'

Reply:
That condition is tested already by test `should throw error if any of the service endpoints in the add-service-endpoints patch is not a valid DID.` DID.ts has 100% code coverage.